plugins/uploadimage/plugin.js

```diff
@@ -146,5 +152,11 @@
 	 * @since 4.5
 	 * @cfg {String} [imageUploadUrl='' (empty string = disabled)]
 	 * @member CKEDITOR.config
+	 *
+	 * Since 4.8.1 version this setting allows to configure custom types for image upload. If not provided default types will be used (jpeg, png, gif, bmp).
```

Comment:
Information about default value and version is provided by `@cfg` and `@since` tags and is redundant in the description.